### PR TITLE
Add repro instructions to fx_graph_runnable.py

### PR DESCRIPTION
**Pull request by zou3519**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #139481

This PR adds some instructions for how to add a TARGETS file to run the
fx_graph_runnable script. I'm planning to add some followups that will
add additional imports for custom ops and use autodeps to get the
dependencies, but I figure this PR is an easy first step.

Test Plan:
- pytest test/dynamo/test_structured_trace.py
- Does anyone have suggestions for how to test this?

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @chenyang78 @kadeng @chauhang @amjames

### CODE REVIEW

File: torch/_dynamo/repro/after_aot.py [hunk 1]
```diff
@@ -225,6 +226,35 @@ def inner_debug_fn(real_inputs):
...
+
+NOTE: you may need additional deps to actually be able to run the script.
+```
+# Contents of TARGETS file
+load("@fbcode_macros//build_defs:python_binary.bzl", "python_binary")
+
+python_binary(
+    name = "repro",
+    main_src = "fx_graph_runnable.py",
+    deps = [
```
Comment (eellison) [inline torch/_dynamo/repro/after_aot.py:247]: could we at least have this include the deps for the operator libraries we print out at the top of the file ? 

https://github.com/pytorch/pytorch/blob/60542eeb33db9ed245fcc47b84d4e530bcdb4e44/torch/_dynamo/debug_utils.py#L48-L55

Right now the file will never run bc we always include those imports.

**Review by eellison — APPROVED** [review]
Looks good.. would still be nice to auto source the right extra deps..

File: torch/_dynamo/repro/after_aot.py (hunk 1 above)
Comment (zou3519) [reply torch/_dynamo/repro/after_aot.py:247]: sounds good!

File: torch/_dynamo/repro/after_aot.py [hunk 2]
```diff
@@ -225,6 +227,38 @@ def inner_debug_fn(real_inputs):
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ #
 
 
+def maybe_fbcode_instructions():
+    if is_fbcode:
+        extra_deps_formatted = "\n".join([f'        "{dep}",' for dep in extra_deps])
+        if len(extra_deps_formatted) > 0:
+            extra_deps_formatted = "\n" + extra_deps_formatted
+        return f"""\
+\"\"\"
```
Comment (XuehaiPan) [inline torch/_dynamo/repro/after_aot.py:236]: nit: just use triple single quotes.

```suggestion
        return f'''\
"""
```

**Pull request closed without merging**